Test/AbstractFilterManagerResultsTest.php

```diff
@@ -19,7 +19,7 @@
 /**
  * This class is able to test results from filters manager.
  */
-abstract class FilterManagerResultsTest extends ElasticsearchTestCase
+abstract class AbstractFilterManagerResultsTest extends ElasticsearchTestCase
```

Comment:
`Test` namespace is maintained and should be supported. 

- [ ] create new file `AbstractFilterManagerResultsTest` with `FilterManagerResultsTest` content 
- [ ] mark `FilterManagerResultsTest` as deprecated.
- [ ] `FilterManagerResultsTest` delete content and extend `AbstractFilterManagerResultsTest`